packages/react/src/components/GroupedList/GroupedListV2.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
```

Comment:
I expected this to be a .tsx file rather than ts.  Generally ts files should be camelCase and tsx files PascalCase to indicate they are components rather than typescript code.

Reply:
Interesting. I can change it but this file uses .ts because I copied over [GroupedList.ts](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/GroupedList/GroupedList.ts).